### ISSUE TITLE
Training Dataset for DIGITS

### ISSUE THREAD

**Issue by yawadugyamfi** [NVIDIA/DIGITS#417]
I am trying to train a deep neural network for recognizing two types of traffic signs: STOP and Yield. I have a database size of about 10000 images. After using Nvidia's digits to train my network, I got an accuracy of about 97%.. However, When I test it against random images like a cat or dog, sometimes I get a 100% prediction that it is a stop sign. 

1. Do you need to supply a deep neural network with a negatives dataset? That's should I have supplied other background information and not just a stop and yield signs.

**Comment by gheinrich** [comment]
If you trained your classifier network to figure out whether an image represents either a STOP or a YIELD sign then the network will be able to do just that. You could add a third class for images that represent neither a STOP nor a YIELD sign, although it sounds as if you would like your network to perform object detection rather than object classification, is it correct?

**Comment by yawadugyamfi** [comment]
Exactly, I am training the system to perform object detection. Reading on how deep learning works, I was expecting that it will be able to learn the unique features of a STOP sign and a Yield sign. By supplying negatives, how different will my system be compared to any simple classification system.

**Comment by lukeyeager** [comment]
Avoiding false positives is a difficult problem, and the standard networks that DIGITS provides may not be particularly well suited for the job. As a first attempt, you can add a third category with a bunch of random images - that may help. For a real system, you may want to investigate using a different learning objective entirely. Sorry for the vague answer - I'm not an expert on this.

I'm closing this issue because it is neither a bug or feature request. The user group is the preferred forum for these types of questions:

https://groups.google.com/d/forum/digits-users

You may also want to ask for help from the Caffe and/or Torch communities:

https://groups.google.com/d/forum/caffe-users
https://groups.google.com/d/forum/torch7
